File: app/src/debug/java/net/chris/demo/guardian/system/DebugApplication.java
package net.chris.demo.guardian.system;

import com.facebook.stetho.Stetho;

import net.chris.tool.debug.library.DebugHelper;

public class DebugApplication extends BaseApplication {

    private DebugHelper debugHelper;

    @Override
    public void onCreate() {
        super.onCreate();
        debugHelper = new DebugHelper();
//        debugHelper.enableStrictMode();
        debugHelper.installLeakCanary(this);
        Stetho.initializeWithDefaults(this);
    }

    @Override
    public void onTerminate() {
        debugHelper.uninstallLeakCanary(this);
        super.onTerminate();
    }
}

File: app/src/debug/kotlin/net/chris/demo/guardian/system/DebugApplication.kt
package net.chris.demo.guardian.system

import com.facebook.stetho.Stetho

import net.chris.tool.debug.library.DebugHelper

class DebugApplication : BaseApplication() {

    private lateinit var debugHelper: DebugHelper

    override fun onCreate() {
        super.onCreate()
        debugHelper = DebugHelper()
        //        debugHelper.enableStrictMode();
        debugHelper.installLeakCanary(this)
        Stetho.initializeWithDefaults(this)
    }

    override fun onTerminate() {
        debugHelper.uninstallLeakCanary(this)
        super.onTerminate()
    }
}
